presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/InternalIcebergConnectorFactory.java

```diff
@@ -63,6 +63,7 @@ public static Connector createConnector(String catalogName, Map<String, String>
                     new HiveAuthenticationModule(),
                     new HiveMetastoreModule(metastore),
                     new MBeanServerModule(),
+                    new IcebergProceduresModule(),
```

Comment:
Not needed here since we install it in `IcebergModule`

Reply:
Removed.